src/gwt/src/org/rstudio/studio/client/workbench/views/terminal/TerminalSession.java

```diff
@@ -740,6 +755,11 @@ public void onError(ServerError error)
          }
       });
    }
+   
+   public void showZombieMessage()
+   {
+      writeln("[Process completed]");
```

Comment:
Would it be useful to show the exit code here? E.g. `[Process completed successfully]` vs. `[Process exited with error code n]`?